pkg/controllers/follower/controller.go

```diff
@@ -353,7 +354,8 @@ func (c *Controller) inferFollowers(
 	sourceGK schema.GroupKind,
 	fedObj fedcorev1a1.GenericFederatedObject,
 ) (sets.Set[FollowerReference], error) {
-	if fedObj.GetAnnotations()[common.EnableFollowerSchedulingAnnotation] != common.AnnotationValueTrue {
+	if fedObj.GetAnnotations()[common.EnableFollowerSchedulingAnnotation] != common.AnnotationValueTrue ||
+		fedObj.GetAnnotations()[common.NoSchedulingAnnotation] == common.AnnotationValueTrue || util.SkipSync(fedObj) {
```

Comment:
I think it needs. For example, assume the workload have been propagated to member clusters, users added the dry-run anno, at this time, dry run will not work, but because we cut off the relationship of leader and follower, follower will not propagate, this will cause the original workload to not work properly.

Reply:
Makes sense. But if this is the case, we probably shouldn't have written the dry-run results into placements, as they can easily be misused by other controllers. An annotation would be better. What do you think?

Reply:
I have tested several situations, and I think they are relatively stable. Placements now only have three controller: global-scheduler, follower-controller and ns-auto controller. 90% of all cases are global-scheduler. Another point is that we are not yet sure whether this restriction `len(fo.status().clusters) == 0`will be removed in the future. This is mainly because there is no such scenario yet. So, I think putting into placement is OK.